clientv3/lease.go

```diff
@@ -98,6 +110,9 @@ type Lease interface {
 	// TimeToLive retrieves the lease information of the given lease ID.
 	TimeToLive(ctx context.Context, id LeaseID, opts ...LeaseOption) (*LeaseTimeToLiveResponse, error)
 
+	// List retrieves all leases.
+	List(ctx context.Context) (*LeaseListResponse, error)
```

Comment:
With this it's possible to do `client.List()`, which isn't clearly related to leases. Right now the pattern for RPCs is rpcname = LeaseX => client name = X. Maybe the RPC should be `LeaseLeases` so the client name can be `Leases()`?